frontend/web/index.html

```diff
@@ -30,6 +27,14 @@
   <link rel="manifest" href="manifest.json">
 </head>
 <body>
+  <script src="https://www.gstatic.com/firebasejs/8.6.1/firebase-app.js"></script>
+  <script>
+    var firebaseConfig = {
```

Comment:
are we missing the firebase config here?

Reply:
I removed it because I initially had the API Key/config stuff here, but I wasn't sure how to store them securely.

Reply:
I thought this might be why it's throwing the no such method error - we could do it in different ways, for example [this](https://medium.com/podiihq/keeping-secret-keys-out-of-version-control-in-flutter-bcd2b1eb9c1b) or [this](https://dev.to/aadityasiva/protecting-api-keys-in-flutter-619).

Reply:
.env looks good to me. I had the config filled with the (I think) correct data. I will try again with a .env file and let you know how it goes